sway-ir/tests/ir_to_ir/bigger_inline.in_ir

```diff
@@ -0,0 +1,54 @@
+// Based on this Sway:
+//
+// script;
+//
+// fn a(b: bool) -> u64 {
+//     let x = 10;
+//     if b {
+//         x
+//     } else {
+//         1
+//     }
+// }
+//
+// fn main() -> u64 {
+//     let mut x = 0;
+//     x = a(true);
+//     x
+// }
+
+script script {
+    fn a(b: bool) -> u64 {
+        local ptr u64 x
+
+        entry:
+        v0 = const u64 10
+        store v0, ptr u64 x
```

Comment:
How come you decided to skip the type of the store value when designing the store instruction? A similar question applies to `cbr`, `phi`, etc. Did you find it easy to guess the type from the context and you wanted to make the IR less verbose?

Reply:
Yeah, I don't think it's needed?  The operand has a type and so we can assume that here that `v0` must be a `u64`.